src/Compilers/Core/MSBuildTaskTests/DotNetSdkTests.cs

```diff
@@ -610,9 +626,13 @@ public void TestGlobalConfigsCanBeManuallyAdded()
 
             VerifyValues(
                 customProps: @"
-<ItemGroup>
+  <PropertyGroup>
+    <!-- Disable automatic global .editorconfig generation by the SDK --> 
+    <GenerateMSBuildEditorConfigFile>false</GenerateMSBuildEditorConfigFile>
+  </PropertyGroup>
```

Comment:
Couldn't we include this in VerifyValues helper for all tests?

Reply:
I feel like being explicit here isn't a bad thing. Certainly, if it became a problem to keep up with, we can extract it in the future.